suite-native/qr-code/src/components/QRCode.tsx

```diff
@@ -58,19 +64,26 @@ export const QRCode = ({
 
     return (
         <>
-            <View style={applyStyle(qrCodeStyle)}>
+            <Box alignItems="center">
                 {data && (
-                    <ReactQRCode
-                        bgColor={colorVariants.standard.backgroundSurfaceElevation1}
-                        fgColor={colorVariants.standard.backgroundNeutralBold}
-                        level="Q"
-                        size={QRCODE_SIZE}
-                        value={data}
-                    />
+                    <View
+                        style={applyStyle(qrCodeContainerStyle, {
+                            backgroundColor: colorVariants.standard.backgroundSurfaceElevation1,
```

Comment:
Why are you passing it as style prop? Why not just place it to style?

Reply:
To make the function pure. There is basically no difference between my and your solution, but I believe that the style function should take input only through its arguments. But should I change it anyway?

Reply:
Function will be pure even if it's used directly because colorVariants is constant.